application/routes/user.py

```
@@ -39,13 +40,17 @@
             email = request.args['email']
+            logger.debug("User: Received registration via GET for user: %s", username)
         else:
             username = request.json['username']
             password = request.json['password']
             email = request.json['email']
+            logger.debug("User: Received registration via POST for user: %s", username)
 
         profiles.add_user(username, password, email)
+        logger.info("User: User registered successfully: %s", username)
     except KeyError as e:
+        logger.error("User: Missing required field during registration: %s", e)
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)